website/orders/services.py

```diff
@@ -26,6 +27,13 @@ def user_is_blacklisted(user):
     return OrderBlacklistedUser.objects.filter(user=user).exists()
 
 
+def user_gets_prioritized_orders(user, shift):
+    """User's order get put first in the queue."""
+    return user in get_users_with_perms(
+        shift.venue, only_with_perms_in=["gets_prioritized_orders_in_venue"], with_superusers=False
+    )
```

Comment:
This will return a queryset so you could optimise this with a .filter(pk=user.pk).exists()